README.md

```diff
@@ -1 +1,205 @@
-# Leaderboard
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <br/>
+
+  <h3><b>README.md</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Leaderboard] <a name="about-project"></a>
+
+**[Leaderboard]** is a website displays scores submitted by different players. It also allows you to submit your score. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- **HTML5**
+- **CSS3**
+- **JavaScript(ES6)**
+- **Webpack**
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Add Task]**
+- **[Remove Task]**
+- **[Edit Task]**
+- **[Display Tasks]**
+- **[Remove All Tasks Checked As Completed]**
+- **[Remove A Selected Task]**
+- **[Using HTML-Webpack-Plugin]**
+- **[Using Webpack CSS-Loader]**
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://zohra-neda.github.io/Leaderboard/dist/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- Hey, Since you won't add a live demo version, let's kindly remove this section from the readme body and from the top navigation of the readme.